eclair-core/src/test/scala/fr/acinq/eclair/blockchain/fee/ConstantFeeProviderSpec.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair.blockchain.fee
+
+import fr.acinq.bitcoin.scalacompat.SatoshiLong
+import org.scalatest.funsuite.AnyFunSuite
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+import scala.concurrent.Await
+
+class ConstantFeeProviderSpec extends AnyFunSuite {
```

Comment:
Adding unit tests for such trivial code is wasteful IMO, we don't need more lines of code if they don't add meaningful value...

Reply:
OK. I removed the ConstantFeeProviderSpec.